app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -36,17 +36,29 @@ class SearchMapper {
         filter.traderName.map("application.holder.businessName" -> nullifyNoneValues(_)) ++
         filter.minDecisionEnd.map("decision.effectiveEndDate" -> greaterThan(_)(formatInstant)) ++
         filter.commodityCode.map("decision.bindingCommodityCode" -> numberStartingWith(_)) ++
-        filter.goodDescription.map("application.goodDescription" -> contains(_)) ++
+        filter.goodDescription.map( gd => any(matchingGoodsDescription(gd) ) ) ++
```

Comment:
I think it would still ready better as either too
`filter.goodDescription.map(description => either(matching("decision.goodDescription", _), matching("application.goodDecrtiption", _))`